src/screens/Report/ReportForm.js

```diff
@@ -41,7 +41,7 @@ const types = [
 ];
 const initialValues = {
   expire: {
-    at: new Date(),
+    at: Date.now() + (1000 * 60 * 60 * 12), // 12 hours from now
```

Comment:
We should extract this as a date utility.

Reply:
I can chuck that somewhere into /utils, maybe group it with other formatting functions (like that header thing?).